proto/controlclient.proto

```diff
@@ -21,21 +21,29 @@ service ScsControl {
   rpc ListIndexes(_ListIndexesRequest) returns (_ListIndexesResponse) {}
 }
 
-message _CreateIndexRequest {
-  message _EuclideanSimilarity{}
+message _SimilarityMetric {
+  message _EuclideanSimilarity {
+  }
```

Comment:
Any reason to separate closing brace } to next line?

Reply:
In the rest of the file, empty messages are closed on the next line. I just glanced at the rest of the file and followed that style.